packages/vsf-storyblok-module/components/global/Img.vue

```diff
@@ -10,22 +10,16 @@
 </template>
 
 <script>
-import { isServer } from '@vue-storefront/core/helpers'
-function canUseWebP () {
-  if (!isServer) {
-    const elem = document.createElement('canvas')
-    if ((elem.getContext && elem.getContext('2d'))) {
-      return elem.toDataURL('image/webp').indexOf('data:image/webp') === 0
-    }
-  }
-  return false
-}
+import { mapGetters } from 'vuex'
 
 export default {
   name: 'StoryblokImage',
   computed: {
+    ...mapGetters({
+      supportsWebp: 'storyblok/supportsWebp'
+    }),
     computedFilters () {
-      if (this.detectWebp && canUseWebP()) {
+      if (this.supportsWebp) {
```

Comment:
Should we use `detectWebp` here to give a choice for developers?
I mean there are pros and cons with webp as with anything. Especially animated webp vs animated gif.
I would suggest we leave the `detectWebp` but keep it as default `true`.
WDYT?